IMC_WebDev/MiniCityBlog.md

```diff
@@ -90,7 +89,7 @@ Now that you have your foundation, you might want to start building. To help you
 <img src=https://lh6.googleusercontent.com/sDm5NTjvOdTJLM17ABgWDYo2vwaXLoNWE1t_8fqJxEBRCHWIuUGPhvsF-SjffJrARWNaDNNJ00awF6usQsISpVBpmnEZtGlhHkIztWAmclhshrbrMZs3m0giMTLE34Cx5K2P87UA>
 
 
-Now that you’ve laid down the foundation and built a road, you can start building more roads, bridges, buildings, or anything else you might want in your mini city! This is what our city looked like: <img src=https://lh4.googleusercontent.com/yu268ztRROLzh3AjLVUeNroSMUlbOsHpZzsWNIokeRtjF7iz-1mqWCqJPdtL1-JyUX6shb3kq28_CYm1xihsf70nPM8EBfHVERg13x7ef5nf5TibItYGI7ixBu_CHcgyIUfRML_c>
+Now that you’ve laid down the foundation and built a road, you can start building more roads, bridges, buildings, or anything else you might want in your mini city! This is what our city looked like after creating all the code described in this tutorial: <img src=https://lh4.googleusercontent.com/yu268ztRROLzh3AjLVUeNroSMUlbOsHpZzsWNIokeRtjF7iz-1mqWCqJPdtL1-JyUX6shb3kq28_CYm1xihsf70nPM8EBfHVERg13x7ef5nf5TibItYGI7ixBu_CHcgyIUfRML_c>
 
 
```

Comment:
You did a really good job taking my suggestions for the text! Did you see the section I wrote regarding your images last time? Did you just decide to leave them as they are? If so, would you mind telling me why?

Reply:
I was unable to figure out how to implement the .svg files into the text. If you strongly insist that I change it I can try to figure it out though!

Reply:
I only suggested SVG files because they retain their qualityno matter how you shrink or enlarge them. As such, they're ideal for diagrams. If they're giving you trouble however, and you don't need that level of quality in your images, you can stick with PNGs or JPGs!

Reply:
@eden-bouskila Emily just used code blocks in MD for her part of the series. In that case, you don't need to use images at all. You can just use the code block tags and copy and paste your code onto your blog. Not only would this make the blog look more professional, but it would make it easy for readers of your blog to just copy and paste code snippets to try them out instead of having to type everything out themselves.